docs/help_with_icg_webform_selfsubmit.md

```diff
@@ -0,0 +1,74 @@
+# Use Case: Islandora Webform for Self-Deposit Workflows
+
+The following is a general overview of what the IW module is capable of and how it is can be used to facilitate self-deposit of electronic theses and dissertations, faculty preprints, or other repository content.
+
+## About the Islandora Webform Module
+
+  * The ICG "Islandora Webform Module" (IW) is a Drupal module developed by Common Media/Born-Digital to add connectivity to an Islandora/Fedora repository.
+  * The IW module is based on the Webform module that is compatible with Drupal 7.
+  * The IW module works on any recent version of Islandora 7x.
+  * The IW module can ingest any kind of object, from PDFs to images, audio, video, manuscripts, and books.
+  * Because the IW module is built on the Drupal Webform module, it inherits all of the latter's functionality. The full functionality can be learned from online documentation and tutorials [see "Documentation" section below].
+  * A working knowledge of MODS, XML Forms, XSLT, and the Fedora object architecture is necessary to set up and manage webforms and workflows.
+
+## Configuring Self-Submit Forms
+
+* Create a new webform (Content > Add Content > Webform)
+
+### Islandora Settings
+Configure how the form interacts with your Islandora collections at webform/islandora. **You must configure these settings before you will be able to map your form components to MODS metadata fields.**
+
+* Select whether to allow the form to show up as a link on certain object pages. The typical self-deposit workflow will likely **not** use this option. 
+* **Content Model Filter:** Select a content model for the objects this form will ingest. Each form can only support one content model, meaning a distinct form will be needed for each media type.
+* **Collection Filter:** Select the collection where submissions will be deposited. Each form can only support deposit to one collection.
+* **Islandora Embargo Settings:** Set global embargo settings for objects submitted via this form or select "Add embargo field to the form" to allow users to set their own embargo. 
+* **Enable Ingest:** Select this option to allow users to upload files. Select the appropriate content model from the Ingest Destination drop-down. Set the relationship of your new object to the collection (typically isMemberOf), and define the appropriate namespace.
```

Comment:
The fact that "current object" refers to the collection selected in the Collection Filter could be clarified here.

Reply:
Chris, I don't see where the text "current object" appears in this file. Can you clarify what you are referring to?

> The fact that "current object" refers to the collection selected in the Collection Filter could be clarified here.

Reply:
TODO - change helptext / labels on https://lasir-demo.born-digital.com/node/14/webform/islandora ("Relationship to current object")